CONTRIBUTING.md

```diff
@@ -0,0 +1,173 @@
+```
+
+## <a name="submit"></a> Submission Guidelines
+
+### <a name="submit-issue"></a> Submitting an Issue
+Help us to maximize the effort we can spend improving the product by not reporting duplicate issues.
+Search the archives before you submit.
+
+Providing the following information will increase the chances of your issue being dealt with quickly:
+
+* **Overview of the Issue** - if an error is being thrown a non-minified stack trace helps
+* **Angular Version** - what version of Angular, Firebase, and AngularFire are you using?
+* **Motivation for or Use Case** - explain what are you trying to do and why the current behavior is a bug for you
+* **Browsers and Operating System** - is this a problem with all browsers?
+* **Reproduce the Error** - provide a live example (using [Plunker][plunker],
+  [JSFiddle][jsfiddle] or [Runnable][runnable]) or a unambiguous set of steps
+* **Related Issues** - has a similar issue been reported before?
+* **Suggest a Fix** - if you can't fix the bug yourself, perhaps you can point to what might be
+  causing the problem (line of code or commit)
+
+You can file new issues by providing the above information [here](https://github.com/angular/angularfire2/issues/new).
+
+### <a name="submit-pr"></a> Submitting a Pull Request (PR)
+
+#### Before you submit:
+
+* Ensure proposed changes or problem have already been clearly defined and
+  discussed in the issue tracker. We don't want you to burn time on code
+  that isn't a good fit for the project.
+* Search [GitHub](https://github.com/angular/angularfire2/pulls) for an open or closed PR
+  that relates to your submission. You don't want to duplicate effort.
+* Please sign our [Contributor License Agreement (CLA)](#cla) before sending PRs.
+  We cannot accept code without this.
+
+#### How to submit:
+
+* Create your patch, **including appropriate test cases**.
+* Follow the [Angular Coding Rules][rules].
+* Run the full test suite, as described in the [developer documentation][dev-doc],
+  and ensure that all tests pass.
+* Commit your changes using a descriptive commit message that follows the
+  [Angular commit message conventions][commit]. Adherence to these conventions
+  is necessary because release notes are automatically generated from these messages.
+
+     ```shell
+     git commit -a
+     ```
+  Note: the optional commit `-a` command line option will automatically "add" and "rm" edited files.
+
+* Push your branch to GitHub:
+
+    ```shell
+    git push origin my-fix-branch
+    ```
+
+* In GitHub, send a pull request to `angular:master`.
+* If we suggest changes then:
+  * Make the required updates.
+  * Re-run the test suites to ensure tests are still passing.
+  * Rebase your branch and force push to your GitHub repository (this will update your Pull Request):
+
+    ```shell
+    git rebase master -i
+    git push -f
+    ```
+
+That's it! Thank you for your contribution!
+
+#### Deploying Docs
+
+1) Build the docs
+
+`$ npm run docs`
```

Comment:
Thanks @davideast. Looking forward to contributing on the docs as they seem a little outdated. 
I read somewhere that I need to request to be added as a contributor to the docs. 
Are there steps I need to follow?
Thanks :)